lib/src/mock_query.dart

```diff
@@ -244,6 +244,16 @@ class MockQuery<T extends Object?> extends FakeQueryWithParent<T> {
     return MockQuery<T>(this, operation);
   }
 
+  dynamic _transformDates(dynamic value) {
+    if(value is DateTime) {
```

Comment:
This is missing a space between `if` and `(`, and wouldn't pass the formatting test workflow. Consider setting up your IDE to format on save to avoid those issues.

Reply:
Well, lately my Android Studio refuses to work with me most of the time, so I can't rely on it too much. Should be ok now.

Reply:
Visual Studio Code works well for me, it formats on Save. Alternatively you could run `dart format .` manually.

https://dart.dev/tools/dart-format